src/constants.ts

```
@@ -304,6 +304,6 @@ export const CAPELLA_FORK_MAPPING: Record<string, string> = {
 };
 
 export const CHAIN_PUBLIC_RPC_URL: Record<number, string> = {
-  1: 'https://ethereum-rpc.publicnode.com', // Mainnet
-  560048: 'https://ethereum-hoodi-rpc.publicnode.com', // Hoodi
+  1:  process.env.RPC_MAINNET || "https://ethereum-rpc.publicnode.com", // Mainnet
```

ESLint

[error] Extra space before value for key '1'.

(@typescript-eslint/key-spacing)